### PR TITLE
Microsoft Entra ID Content Pack

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CRTX-120008)

## Description
This pack add modeling rules and time parsing for Microsoft Entra ID (former known as AAD).

### CODE REVIEW

File: Packs/MicrosoftEntraID/README.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+<html>
+
+<h1>Microsoft Entra ID</h1>
+
+<details>
+    <summary><h2 style="display:inline-block">What does this pack do</h2></summary>
+
+<h3>Log Normalization - One Data Model</h3>
+<p>This pack support normalization of the below log categories of Microsoft Entra ID:</p>
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:11]:
```suggestion
<p>This pack supports normalization of the below log categories of Microsoft Entra ID:</p>
```

File: Packs/MicrosoftEntraID/README.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+<ol>
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:12]: This should be a bulleted list.  Numbered lists are used when you have to do something in a particular order such as instructions.

File: Packs/MicrosoftEntraID/README.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    <li>ADFSSignInLogs</li>
+    <li>ProvisioningLogs</li>
+    <li>RiskyUsers</li>
+    <li>UserRiskEvents</li>
+    <li>RiskyServicePrincipals</li>
+    <li>ServicePrincipalRiskEvents</li>
+</ol>
+
+<h3>Timestamp Parsing</h3>
+<p>Timestamp parsing relies on 2 fields, which depends on the log category:</p>
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:27]: Should this be 3 fields?

File: Packs/MicrosoftEntraID/README.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+<ol>
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:28]: These should also be a bulleted list.

File: Packs/MicrosoftEntraID/README.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+    </ol>
+  </li>
+</ol>
+</details>
+<hr>
+
+<details>
+<summary><h2 style="display:inline-block">Data Collection</h2></summary>
+<h3 >Entra ID Side</h3>
+<p>To configure Microsoft Entra ID to send logs to XSIAM, follow the below steps.</p>
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:59]:
```suggestion
<p>To configure Microsoft Entra ID to send logs to Cortex XSIAM, follow the below steps.</p>
```

File: Packs/MicrosoftEntraID/README.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+<h4>Prerequisites</h4>
+<ol>
+    <li>Create an <b>Azure event hub</b>. For more information, refer to Microsoft's official <a href="https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create">documetaion</a>.</li>
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:62]:
```suggestion
    <li>Create an <b>Azure event hub</b>. For more information, refer to Microsoft's official <a href="https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create">documentation</a>.</li>
```

File: Packs/MicrosoftEntraID/README.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+    <li>Make sure that you have at least Security Administrator role.</li>
+</ol>
+<h4>Stream logs to an event hub</h4>
+<ol>
+    <li>Sign in to the <b>Microsoft Entra admin center</b>.</li>
+    <li>Navigate to <b>Identity</b> &rarr; <b>Monitoring & health</b> &rarr; <b>Diagnostic settings</b>.</li>
+    <li>Select <b>+ Add diagnostic setting</b> to create a new integration or select <b>Edit setting</b> for an existing integration.</li>
+    <li>Enter a <b>Diagnostic setting name</b>. If you're editing an existing integration, you can't change the name.</li>
+    <li>Select the log categories that you want to stream. Refer to the <b>Log Normalization</b> section for the supported log categories for normalization.</li>
+    <li>Select the Stream to an event hub check box.</li>
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:72]:
```suggestion
    <li>Select the <b>Stream to an event hub </b>checkbox.</li>
```

File: Packs/MicrosoftEntraID/README.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+    <li>Select the Azure subscription, Event Hubs namespace, and optional event hub where you want to route the logs.</li>
+</ol>
+
+<p>For more information, refer to Microsoft's official <a href="https://learn.microsoft.com/en-us/entra/identity/monitoring-health/howto-stream-logs-to-event-hub">documetaion</a>.</p>
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:76]:
```suggestion
<p>For more information, refer to Microsoft's official <a href="https://learn.microsoft.com/en-us/entra/identity/monitoring-health/howto-stream-logs-to-event-hub">documentation</a>.</p>
```

File: Packs/MicrosoftEntraID/README.md [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+
+<h3>XSIAM side</h3>
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:78]:
```suggestion
<h3>Cortex XSIAM side</h3>
```

File: Packs/MicrosoftEntraID/README.md [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+<p>To connect XSIAM to the Azure Event Hub, follow the below steps.</p>
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:79]:
```suggestion
<p>To connect Cortex XSIAM to the Azure Event Hub, follow the below steps.</p>
```

**Comment by ShirleyDenkberg** [comment]
@cweltPA @OBavly Doc review completed.

File: Packs/MicrosoftEntraID/README.md (hunk 3 above)
Comment (yasta5) [reply Packs/MicrosoftEntraID/README.md:27]: You are right.

File: Packs/MicrosoftEntraID/README.md (hunk 2 above)
Comment (yasta5) [reply Packs/MicrosoftEntraID/README.md:12]: Changed it.

File: Packs/MicrosoftEntraID/README.md (hunk 4 above)
Comment (yasta5) [reply Packs/MicrosoftEntraID/README.md:28]: changed.

**Review by OBavly — APPROVED** [review]

File: Packs/MicrosoftEntraID/README.md [hunk 11]
```diff
@@ -16,10 +16,41 @@
...
 - YYYY-MM-DDTH:M:S.S* (UTC)
 
+***
+
+## Data Collection
+To configure Microsoft Entra ID to send logs to Cortex XSIAM, follow the below steps.
+
+### Prerequisites
+- Create an **Azure event hub**. For more information, refer to Microsoft's official [documentation](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create).
+- Make sure that you have at least Security Administrator role.
```
Comment (OBavly) [inline Packs/MicrosoftEntraID/README.md:30]: at least *a* security administrator role

File: Packs/MicrosoftEntraID/README.md [hunk 12]
```diff
@@ -16,10 +16,41 @@
...
+
+### Stream logs to an event hub
+1. Sign in to the **Microsoft Entra admin center**.
```
Comment (OBavly) [inline Packs/MicrosoftEntraID/README.md:33]: Capitalize Admin Center

File: Packs/MicrosoftEntraID/README.md [hunk 13]
```diff
@@ -16,10 +16,41 @@
...
+2. Navigate to **Identity** &rarr; **Monitoring & health** &rarr; **Diagnostic settings**.
```
Comment (OBavly) [inline Packs/MicrosoftEntraID/README.md:34]: Capitalize Health and Settings

File: Packs/MicrosoftEntraID/README.md [hunk 14]
```diff
@@ -16,10 +16,41 @@
...
+3. Select **+ Add diagnostic setting** to create a new integration or select **Edit setting** for an existing integration.
```
Comment (OBavly) [inline Packs/MicrosoftEntraID/README.md:35]: Capitalize if the options are capitalized in the official page

File: Packs/MicrosoftEntraID/README.md [hunk 15]
```diff
@@ -16,10 +16,41 @@
...
+7. Select the Azure subscription, Event Hubs namespace, and optional event hub where you want to route the logs.
+
+For more information, refer to Microsoft's official [documentation](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create">documentation).
+
+### Cortex XSIAM side
+To connect Cortex XSIAM to the Azure Event Hub, follow the below steps.
+
+#### Azure Event Hub Collector
+1. Navigate to **Settings** &rarr; **Data Sources**.
+2. If you have already configured an **Azure Event Hub Collector**, select the **3 dots**, and then select **+ Add New Instance**. If not, select **+ Add Data Source**, search for "Azure Event Hub" and then select **Connect**.
```
Comment (OBavly) [inline Packs/MicrosoftEntraID/README.md:48]: Maybe turn "3 dots" to "...", or rather "select the '...' symbol"?

File: Packs/MicrosoftEntraID/README.md (hunk 13 above)
Comment (yasta5) [reply Packs/MicrosoftEntraID/README.md:34]: It is with small letters in both ms docs and the portal itslef.

File: Packs/MicrosoftEntraID/README.md (hunk 12 above)
Comment (yasta5) [reply Packs/MicrosoftEntraID/README.md:33]: It is with small letters in both ms docs and the portal itself.

File: Packs/MicrosoftEntraID/README.md (hunk 14 above)
Comment (yasta5) [reply Packs/MicrosoftEntraID/README.md:35]: Not capitalized in the official page

File: Packs/MicrosoftEntraID/README.md (hunk 15 above)
Comment (yasta5) [reply Packs/MicrosoftEntraID/README.md:48]: Vertical dots.

**Pull request merged**